tests/python/relay/test_op_level5.py

```diff
@@ -393,8 +393,6 @@ def verify_nms(
             intrp2 = relay.create_executor("debug", ctx=ctx, target=target)
             op_res2 = intrp2.evaluate(func)(x0_data, x1_data, x2_data, x3_data)
             tvm.testing.assert_allclose(op_res2.asnumpy(), ref_res, rtol=1e-5)
-            if target == "nvptx":
```

Comment:
I think I removed this condition in https://github.com/apache/tvm/pull/7051 
Why is this diff here? Need rebase?

Reply:
I haven't rebased since Monday, will fix